wiki/Tournaments/UKCC/UKCC8/en.md

```diff
@@ -0,0 +1,163 @@
+---
+tags:
+  - UK Community Cup 8
+  - UKCC 8
+  - UKCC 8
+---
+
+# UK Community Cup 8
+
+![UKCC 8 logo](img/banner.png)
+
+The 8th edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition is a 2v2.
+
+Current Stage: **[Signups (click for signup form)](https://forms.gle/s1FQ1sRTrwXa4i8t8)**
```

Comment:
this isn't rly needed since the available info on the page already implies what stage the tourney's at. also it may become out of date if you write it like this

Reply:
Plan was to update it (as have done now we're not in Signups) but can remove it 😄